astro/src/content/quickstarts/quickstart-javascript-express-api.mdx

```diff
@@ -0,0 +1,335 @@
+
+You'll find a Docker Compose file `docker-compose.yml` and an environment variables configuration file `.env` in the root folder of the repository.
+
+Assuming you have Docker installed on your machine, you can start FusionAuth on your machine with the following.
+
+```
+docker compose up -d
+```
+
+This will start three containers, one each for FusionAuth, Postgres, and Elasticsearch.
+
+Here you are using a bootstrapping feature of FusionAuth, called [Kickstart](/docs/v1/tech/installation-guide/kickstart). When FusionAuth starts for the first time, it will look at the `kickstart/kickstart.json` file and configure FusionAuth to your specified state.
+
+<Aside type="note">
+If you ever want to reset the FusionAuth system, delete the volumes created by `docker compose` by executing `docker compose down -v`, then rerun `docker compose up -d`.
+</Aside>
+
+FusionAuth will be initially configured with these settings:
+
+* Your client Id is: `e9fdb985-9173-4e01-9d73-ac2d60d1dc8e`.
+* Your client secret is: `super-secret-secret-that-should-be-regenerated-for-production`.
+* Your example teller username is `teller@example.com` and your password is `password`. They have the role `teller`.
+* Your example customer username is `customer@example.com` and your password is `password`. They have the role `customer`.
+* Your admin username is `admin@example.com` and your password is `password`.
+* Your fusionAuthBaseUrl is 'http://localhost:9011/'.
+
+You can log in to the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don't need to.
+
+<Aside type="caution">
+  The `.env` and `kickstart.json` files contain passwords. In a real application, always add these files to your `.gitignore` file and never commit secrets to version control.
+</Aside>
+
+### Create A Basic {frontmatter.technology} Application
+
+Now you are going to create an {frontmatter.technology} API application. While this section builds a simple API, you can use the same configuration to integrate an existing API with FusionAuth.
+
+We are going to be building an API backend for a banking application called ChangeBank. This API will have two endpoints:
+- `make-change`: This endpoint will allow you to call GET with a `total` amount and receive a response indicating how many nickels and pennies are needed to make change. Valid roles are `customer` and `teller`.
+- `panic`: Tellers may call this POST endpoint to call the police in case of an incident. The only valid role is `teller`.
+
+Both endpoints will be protected such that a valid JSON web token (JWT) will be required in a cookie in order to be accessed. Additionally, the JWT must have a `roles` claim containing the appropriate role to use the endpoint.
+
+Create the starter app called `mysite` for this API.
+
+```shell
+npx express-generator@latest mysite --no-view
+cd mysite
+npm install
+npm audit fix --force # run this only if npm reports security vulnerabilities
```

Comment:
@mark-robustelli - this is the official express generator, and the problem is with its dependencies. Unfortunately this is a common problem for node packages - a workaround could be to commit the code it generates but this would get out of date and then the user wouldn't be able to start 'from scratch', but we'd recommend just leaving in the 'fix' step for now until the dependencies are fixed.